var/spack/repos/builtin/packages/py-ci-info/package.py

```diff
@@ -0,0 +1,22 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class PyCiInfo(PythonPackage):
+    """Continuous Integration Information
```

Comment:
```suggestion
    """Continuous Integration Information.
```
`spack info` will remove newline chars, so it would end up being a run-on sentence without punctuation.